### PR TITLE
Feature/add overview

### PR DESCRIPTION
**Pull request by jtsoi**
@butschi @jacquelinend Please take a look.

### CODE REVIEW

File: source/integration-guide/overview.rst [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+------------
+At this moment, the integration has only one dependency, jQuery 1.3 or later.
+If jQuery is not found on the page, the integration script will asynchronously
+load jQuery from the Google CDN (this will not slow down the page as it is done
+asynchronously and most client browsers already have it cached).
+
+Integration script hosting
+--------------------------
+The integration script is hosted on the global Amazon CDN CloudFront. The file itself has
+a sane Cache-Control header values set to ensure the client browser checks for updated versions
```
Comment (jacquelinend) [inline source/integration-guide/overview.rst]: values -> value?

Comment (jacquelinend) [inline source/integration-guide/overview.rst]: ensure the -> ensure that the

File: source/integration-guide/overview.rst [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+
+**Step 2**
+
+As soon as the first line of the snippet executes, the snippet creates an array,
+and all calls to the skeleton API are recorded in this "API calls array".
+Besides providing the skeleton JS API, the snippet triggers the load of the integration script.
+
+The integration script is loaded by inserting a <script> tag into the head element (using insertBefore call),
+it is the most common way to include third-party scripts, and ensure async loading.
```
Comment (jacquelinend) [inline source/integration-guide/overview.rst]: ), it is the most common way to include third-party scripts, and ensure async loading. ->
); this is the most common way to include third-party scripts and ensure async loading.

File: source/integration-guide/overview.rst (hunk 1 above)
Comment (jtsoi) [inline source/integration-guide/overview.rst]: OK

**Pull request merged**